static/sass/_v1_pattern_card.scss

```diff
@@ -38,4 +38,22 @@
       text-decoration: underline;
     }
   }
+
+  .p-strip--accent {
+    .p-card--highlighted {
+      color: $color-mid-dark;
+
+      a:link,
+      a:visited {
+        color: $color-brand;
+      }
+
+      a:hover,
+      a:active {
+        color: $color-brand;
+        text-decoration: underline;
+      }
```

Comment:
Can you make sure to file an issue upstream for this and comment this with a XXX and a link to the issue.